bitsandbytes/backends/cpu_xpu_common.py

```diff
@@ -562,7 +562,7 @@ def gemm_4bit_impl(
             None,
             None,
             state.blocksize,
-            ipex_cpu.quantization.WoqLowpMode.BF16,
+            2,
```

Comment:
Feels like there should be a comment about the magic number "2".

Reply:
fixed